### ISSUE TITLE
Bilinear filtering?

### ISSUE THREAD

**Issue by Anths2016** [npwoods/bletchmame#55]
It appears that bletchmame sets bilinear filtering, without the possibility of modification.

**Comment by npwoods** [comment]
Yes, BletchMAME doesn't surface all of the bells and whistles that MAME has.  Is it possible for you to identify the MAME options that you would want to see surfaces (e.g. - a command line?).

I should also note that the MAME facility to attach to a window doesn't work with some of the stuff you can do on MAME.

**Comment by randomuser4444** [comment]
It's probally a bit early yet! but hey only as a suggestion later on, it would be cool to have basic some basic upscaling options and at one point hooking in shaders something like CRT-Geom-Deluxe that MAME currently already uses.

**Comment by npwoods** [comment]
Yeah I'll flat out acknowledge that I have not really been a consumer of all of the video-related bells and whistles in MAME, and I'm not in a good position to design how they would integrate into BletchMAME.  Furthermore, there is a very real possibility that some of these options in MAME are not compatible with `-attach_window`.

If anybody is up for refining a specific proposal, I'm very open to supporting it.
